### ISSUE TITLE
How to use DiagGaussianDistribution? the default is CategoricalDistribution

### ISSUE THREAD

**Issue by donotbelieveit** [sjtu-marl/malib#61]
Hi, I am training my multi-agent  environment (continuous action, continuous observation) with PSRO+PPO in branch "policy-support-baseline", see the paper [Emergent Complexity via Multi-agent Competition](https://arxiv.org/abs/1710.03748) for the specific environment. I found that  in Policy's probability distribution, if the action space is continuous, a DiagGaussianDistribution is returned. see [distribution.py](https://github.com/sjtu-marl/malib/blob/policy-support-baseline/malib/common/distributions.py) ,line 876. But here on line 125 of [malib/rl/pg/policy.py](https://github.com/sjtu-marl/malib/blob/policy-support-baseline/malib/rl/pg/policy.py) the default here is to use CategoricalDistribution (because only proba_distribution of the CategoricalDistribution class has the action_mask parameter),see the figure below. How do I use DiagGaussianDistribution? Can the logits obtained in the figure, line 122, help me to use it? I am looking forward to read your answer, thank you!

<img width="749" alt="截屏2023-03-09 上午11 15 40" src="https://user-images.githubusercontent.com/49392787/223907537-96324b90-0d19-40fc-af33-e761ffe8f543.png">

**Comment by KornbergFresnel** [comment]
@donotbelieveit a good question, you can try to override this function with DiagGaussianDistribution as your `dist_fn`

**Comment by donotbelieveit** [comment]
how do I get parameters "mean_actions" and "log_std" that DiagGaussianDistribution need?is there any suggestions

**Comment by KornbergFresnel** [comment]
> how do I get parameters "mean_actions" and "log_std" that DiagGaussianDistribution need?is there any suggestions

the `mean_actions` and `log_std` should be the outputs our your policy outputs.

**Comment by donotbelieveit** [comment]
thank you for replying me!I am a second year student in UCAS,looking forward to having more communication with you about the use of the framework.
